azurerm_api_management/output.tf

```diff
@@ -1,4 +1,4 @@
-output "id" {
-  description = "The Azure resource group id"
-  value       = "${module.azurerm_api_management.id}"
+output "name" {
```

Comment:
Please use azurerm_api_management instead

Reply:
the output from the arm template cannot be retrieved